### DISCUSSION TITLE
Cross repository PR using REST API

### DISCUSSION THREAD

**Discussion by parteekcoder** [community/community#56779] [Apps, API and Webhooks]
### Hi everyone,

Hi,

In my application, I want to create a pull request from one repo(forked) to its parent repo. How can I do that using Github REST API if both these repos are on different accounts. I am unable to find the example which explains how this done across different repos. All the examples explain it within single organization.

Could you please help me to figure it out

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @parteekcoder , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by parteekcoder** [reply]
Thanks @samus-aran
